src/gui/nvim-qt.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Neovim
```

Comment:
Probably better to go with Neovim-Qt here. Some people have wrappers for launching term+Neovim.